src/utils/promisifyTypeField.js

```diff
@@ -0,0 +1,17 @@
+import { promisifyObjectValue } from './promisifyObjectValue';
+
+/**
+ * Wraps a field value of object in Promise if it needed by scheme
+ *
+ * @param {any} valueFromObject value from object
+ * @param {import('../../types').Scheme} valueFromScheme scheme of field value
+ * @param {import('../../types').Callback} [mapAfterCallback] mapAfter callback from scheme
+ * @returns {any}
+ */
+export function promisifyTypeField(valueFromObject, valueFromScheme, mapAfterCallback = arg => arg) {
```

Comment:
I suggest to make signature same as `promisifyObjectValue` and to have two arguments here:
```js
export function promisifyTypeField(valueFromObject, fieldScheme)
```

Reply:
Ok